blockchain/asm/doc.go

```diff
@@ -1,22 +1,27 @@
-// Copyright 2018 The klaytn Authors
-// Copyright 2017 The go-ethereum Authors
-// This file is part of the go-ethereum library.
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
 //
-// The go-ethereum library is free software: you can redistribute it and/or modify
+// The klaytn library is free software: you can redistribute it and/or modify
```

Comment:
What makes you change this license statement?

Reply:
"doc.go" file doesn't exist in https://github.com/ethereum/go-ethereum/tree/master/core/asm

Reply:
I changed back the copyright since a package description is originated from go-ethereum.